javaslang/generator/Generator.scala

```diff
@@ -1558,6 +1569,19 @@ def generateTestClasses(): Unit = {
         """
       }
 
+      def genExtAliasTest(name: String, func: String, value: String, check: String): String = {
+        xs"""
+          @$test
+          public void should${name}() {
+              assertThat($func($value)).$check;
+          }
+        """
+      }
+
+      def genMediumAliasTest(name: String, func: String, value: String): String = genExtAliasTest(name + "ReturnNotNull", func, value, "isNotNull()")
```

Comment:
I don't mind either way, but since you used them previously, I thought it could be more consistent if you unify it. Whichever you prefer, I'm just expressing observations, the choice is yours :)